scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -56,15 +56,22 @@ def test_rel_entr_generic(dtype):
 
 @pytest.mark.fail_slow(5)
 # `reversed` is for developer convenience: test new function first = less waiting
-@pytest.mark.parametrize('f_name,n_args', reversed(array_special_func_map.items()))
+@pytest.mark.parametrize(
+        'f_name,n_args',
+        reversed([(name, info.n_args)
+                  for name, info in array_special_func_map.items()]))
 @pytest.mark.filterwarnings("ignore:invalid value encountered:RuntimeWarning:dask")
 @pytest.mark.parametrize('dtype', ['float32', 'float64'])
 @pytest.mark.parametrize('shapes', [[(0,)]*4, [tuple()]*4, [(10,)]*4,
                                     [(10,), (11, 1), (12, 1, 1), (13, 1, 1, 1)]])
 def test_support_alternative_backends(xp, f_name, n_args, dtype, shapes):
+    if f_name in {'expn', 'polygamma'}:
+        pytest.skip(f"`{f_name}` has limited domain.")
+    if f_name in {'multigammaln'}:
+        pytest.skip(f"`{f_name}` requires some integer arguments.")
```

Comment:
It would be _very_ good to get https://github.com/scipy/scipy/pull/22718 in so that we can think how to build some tooling for these on top of it.

Reply:
Agreed.  Should I wait for that PR?

Reply:
I think it's ok to leave these untested for the time being. Just know that the more changes you do to the test module the more either of us will have to rework to fix a conflict.